src/react-apps/applications/service-development/src/features/deploy/deploySagas.ts

```diff
@@ -12,27 +13,9 @@ export function* fetchDeploymentsSaga({
   repo,
 }: DeployActions.IFetchDeployments): SagaIterator {
   try {
-    // TODO: TLS is missing on cluster. Update after TLS is available.
-    // env = 'at22';
-    // const result = yield call(get,
-    //   `http://${org}.apps.${env}.altinn.cloud/kuberneteswrapper/deployments?labelSelector=release=${org}-${repo}`);
-
-    const result = [
-      {
-        spec: {
-          template: {
-            spec: {
-              containers: [
-                {
-                  name: 'example-repo',
-                  image: 'tddregistry.azurecr.io/tdd-helloworld:1',
-                },
-              ],
-            },
-          },
-        },
-      },
-    ];
+    const result = yield call(get,
+      // tslint:disable-next-line:max-line-length
```

Comment:
Not related to your PR, I just notice that we disable this rule very often, might be good to just remove it all together?

Reply:
Maybe, please discuss at tech forum :)